.eslintrc

```diff
@@ -1,41 +1,47 @@
 {
-  "extends": "react-app",
-  "parser": "babel-eslint",
+  "extends": ["eslint:recommended", "plugin:react/recommended"],
```

Comment:
A lot of the configurations had to be updated due to babel switching over to a monorepo format. I've switched some of the rulesets to warnings (default was error) since the configuration has caught a lot of linting issues in the whole codebase. Erroring out will just prevent the app from running. These rules should be adjusted over time to fit our code standards.